src/main/scala/tech/cryptonomic/conseil/Lorre.scala

```diff
@@ -80,16 +87,40 @@ object Lorre extends App with TezosErrors with LazyLogging with LorreAppConfig w
     apiOperations
   )
 
-  /** Inits registered tokens at startup */
+  /** Inits tables with values from CSV files */
+  import kantan.csv._
   import kantan.csv.generic._
 
-  val tokenRows: List[Tables.RegisteredTokensRow] =
-    ConfigUtil.Csv.readTableRowsFromCsv(Tables.RegisteredTokens, tezosConf.network, separator = '|')
+  implicit val timestampDecoder: CellDecoder[java.sql.Timestamp] = (e: String) => {
+    val format = DateTimeFormatter.ofPattern("EEE MMM dd yyyy HH:mm:ss 'GMT'Z (zzzz)")
+    StringDecoder
+      .makeSafe("Instant")(s => Instant.from(format.parse(s)))(e)
+      .map(Timestamp.from)
+      .left
+      .map(x => DecodeError.TypeError(x.message))
+  }
```

Comment:
This will be impacted by possible changes to #733, I assume.

I suggest putting this inside the initTable.. method rather than here.

Reply:
I cannot. Because I use type parameters in the method it ti needs to have implicits for the defined types beforehand. There is now way of generating decoders for a type `A`, but before method is being called the type is known and this way it can work.